react/Panel/styles.styl

```diff
@@ -0,0 +1,10 @@
+@require '../../stylus/components/panels'
+
+.Panel-group
```

Comment:
Is the nesting really necessary? It doesn't do much besides inscreasing the specificity I think.

Reply:
You're right.